src/update-feeds/dapi-data-registry.ts

```diff
@@ -24,6 +26,31 @@ export const decodeDapisCountResponse = (dapiDataRegistry: DapiDataRegistry, dap
 
 export type DapisCountResponse = ReturnType<typeof decodeDapisCountResponse>;
 
+export const decodeDataFeed = (dataFeed: string): DecodedDataFeed => {
+  if (dataFeed.length === 130) {
```

Comment:
I think it would be nice to add a permalink to the contract to explain this encoding/decoding beahviour.